package.json

```diff
@@ -38,7 +38,7 @@
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
     "assert": "^2.0.0",
-    "chromedriver": "^114.0.0",
+    "chromedriver": "^115.0.1",
```

Comment:
116 is latest https://www.npmjs.com/package/chromedriver and that's what I'm seeing when the CI installs the deb.

Reply:
Are we looking at the same page? It say 115.0.1 is the latest tag and it corresponds to what is installed when run: `npm install chromedriver@latest`

It's true that the latest chrome (not the npm package) is 116 on the stable version: https://googlechromelabs.github.io/chrome-for-testing/

Reply:
Double checked: It could be that the changes weren't propagated to npm yet -- there was actually a release on the github repo for 116 after I pushed these changes: https://github.com/giggio/node-chromedriver/tags

I'll be happy to update this after checks pass